system/keymaps/customcontroller.SiriRemote.xml

```diff
@@ -128,7 +128,7 @@
   </VideoMenu>
   <Videos>
     <customcontroller name="SiriRemote">
-      <button id="7">Info</button>
+      <button id="7">ContextMenu</button>
```

Comment:
since `button 7` is already mapped to `ContextMenu` in the global section, you don't need to re-define it here again.
you can simply remove the entire Videos section.

Reply:
Ah yes indeed, I did not think of that. Thank you

Reply:
Hum, if I remove the entire videos section Context menu doesn't work.

Reply:
Nothing I've looked at. My long term plan is to get rid of the usage of the custom controller config. I'd honestly just leave it how you have, better things to spend your time on, but your choice

Reply:
FYI only deleting the `<button ...` works fine: https://forum.kodi.tv/showthread.php?tid=354965&pid=2972076#pid2972076